scripts/scil_filter_tractogram.py

```diff
@@ -133,20 +140,28 @@ def main():
     if args.verbose:
         logging.basicConfig(level=logging.DEBUG)
 
-    roi_opt_list = prepare_filtering_list(parser, args)
+    roi_opt_list, only_filtering_list = prepare_filtering_list(parser, args)
+    o_dict = {}
 
     sft = load_tractogram_with_reference(parser, args, args.in_tractogram)
 
     # Streamline count before filtering
-    sc_bf = len(sft.streamlines)
+    o_dict['streamline_count_before_filtering'] = len(sft.streamlines)
 
     for i, roi_opt in enumerate(roi_opt_list):
+        curr_dict = {}
         # Atlas needs an extra argument (value in the LUT)
         if roi_opt[0] == 'atlas_roi':
             filter_type, filter_arg, filter_arg_2, \
                 filter_mode, filter_criteria = roi_opt
         else:
             filter_type, filter_arg, filter_mode, filter_criteria = roi_opt
+
+        curr_dict['Filename'] = os.path.abspath(filter_arg)
```

Comment:
Are you sure about the upper case ? I personally don't like it, I prefer all lower case (optional)